scripts/verify-manifest.sh

```
@@ -0,0 +1,151 @@
+module_dir=$3
+
+root_dir=$(git rev-parse --show-toplevel)
+validate_params "$env" "$target_org" "$module_dir"
+
+# Create a temporary directory for output
+temp_dir=$(mktemp -d)
+output_file="${temp_dir}/modified_files.txt"
+
+trap cleanup EXIT
+
+list_apex_tests_script="${root_dir}/scripts/list-apex-tests.mjs"
```

Shellcheck (0.10.0)

[warning] list_apex_tests_script appears unused. Verify use (or export if used externally).

(SC2034)